frontend/.eslintrc

```diff
@@ -33,7 +39,6 @@
         "jsx-a11y/click-events-have-key-events": ["off"],
         "jsx-a11y/interactive-supports-focus": ["off"],
         "quotes": ["error", "double"],
-        "no-shadow": ["error", { "allow": ["_"]}],
```

Comment:
😢 i like this rule

Reply:
i don't

Reply:
smh

Reply:
yea oof had to remove this rule because it conflicts with typescript enums :(

Reply:
fyi, this should should be already included in the default eslitn config for the ts version